pygments/lexers/arturo.py

```diff
@@ -0,0 +1,250 @@
+"""
+    pygments.lexers.arturo
+    ~~~~~~~~~~~~~~~~~~~~~~
+
+    Lexer for the Arturo language.
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+
+from pygments.lexer import bygroups, default, DelegatingLexer, do_insertions,\
+    include, RegexLexer, this, using, words
+from pygments.token import Comment, Generic, Keyword, Name, Number, Operator,\
+    Other, Punctuation, String, Text
+
+from pygments.util import ClassNotFound, get_bool_opt
+
+__all__ = [
+    'ArturoLexer'
+]
+class ArturoLexer(RegexLexer):
+    """
+    For Arturo source code
+
+    [Arturo's Github](https://github.com/arturo-lang/arturo)
+    [Arturo's Website](http://arturo-lang.io/)
```

Comment:
This is reST markup, not Markdown...

Reply:
- [x] [change hyperlinks to reST format.](https://github.com/pygments/pygments/pull/2259/commits/52ef9c821fcfb5925042374f62dd7b2e1d15dd78)